modules/psap-special-resource-operator-using-CRD.adoc

```diff
@@ -0,0 +1,404 @@
+                                        scheduler.alpha.kubernetes.io/critical-pod: ""
+                                    labels:
+                                        app: {{ printf "%s-%s" .Values.specialResourceModule.metadata.name .Values.kernelFullVersion | replace "." "-" | replace "_" "-" | trunc 63 }}
+                                spec:
+                                    serviceAccount: {{.Values.specialResourceModule.metadata.name}}
+                                    serviceAccountName: {{.Values.specialResourceModule.metadata.name}}
+                                    containers:
+                                    - image: {{.Values.registry}}/{{.Values.specialResourceModule.metadata.name}}-{{.Values.groupName.driverContainer}}:{{.Values.kernelFullVersion}}
+                                      name: {{.Values.specialResourceModule.metadata.name}}
+                                      imagePullPolicy: Always
+                                      command: [sleep, infinity]
+                                      lifecycle:
+                                        preStop:
+                                            exec:
+                                                command: ["modprobe", "-r", "-a" , "simple-kmod", "simple-procfs-kmod"]
+                                      securityContext:
+                                        privileged: true
+----
+.. Save this YAML template for the placement of policies in the `templates` directory as `0003-policy.yaml`.
++
+[source,yaml]
+----
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+    name: {{.Values.specialResourceModule.metadata.name}}-placement
+spec:
+    clusterConditions:
+    - status: "True"
+      type: ManagedClusterConditionAvailable
+    clusterSelector:
+      matchExpressions:
+      - key: name
+        operator: NotIn
+        values:
+        - local-cluster
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+    name: {{.Values.specialResourceModule.metadata.name}}-binding
+placementRef:
+    apiGroup: apps.open-cluster-management.io
+    kind: PlacementRule
+    name: {{.Values.specialResourceModule.metadata.name}}-placement
+subjects:
+    - apiGroup: policy.open-cluster-management.io
+      kind: Policy
+      name: policy-{{.Values.specialResourceModule.metadata.name}}-ds
+----
++
+.. Change into the `charts/acm-simple-kmod-0.0.1` directory:
++
+[source,terminal]
+----
+cd ..
+----
+.. Save the following YAML template for the chart as `Chart.yaml` in the `charts/acm-simple-kmod-0.0.1` directory:
++
+[source,yaml]
+----
+apiVersion: v2
+name: acm-simple-kmod
+description: Build ACM enabled simple-kmod driver with SpecialResourceOperator
+icon: https://avatars.githubusercontent.com/u/55542927
+type: application
+version: 0.0.1
+appVersion: 1.6.4
+----
+. From the `charts` directory, create the chart using the `helm package` command:
```

Comment:
(Optional, suggesting because it says `helm package` immediately after and we don't usually put the command name in the step)
```suggestion
. From the `charts` directory, create the chart using the following command:
```